draft-packages/form/KaizenDraft/Form/CheckboxGroup/README.mdx

```diff
@@ -14,7 +14,7 @@ health:
   designed: true
   documented: true
   implemented: true
-  latestDesign: false
+  latestDesign: true
```

Comment:
If the error state is missing, does it still count as meeting the latest design?

Reply:
That would be considered as part of allVariants, so maybe I should go to false on that.